build.sbt

```diff
@@ -100,22 +115,13 @@ lazy val docs = (project in file("docs"))
       import laika.config.SyntaxHighlighting
       Seq(Markdown.GitHubFlavor, SyntaxHighlighting)
     },
+    publish / skip := true,
     Laika / sourceDirectories := Seq((ThisBuild / baseDirectory).value / "docs")
   )
 
-credentials += Credentials(Path.userHome / ".sbt" / "sonatype_credentials")
-
-licenses := Seq("MIT" -> url("http://opensource.org/licenses/MIT"))
-homepage := Some(url("https://github.com/mrpowers-io/spark-fast-tests"))
-developers ++= List(
-  Developer("MrPowers", "Matthew Powers", "@MrPowers", url("https://github.com/MrPowers"))
-)
 scmInfo := Some(ScmInfo(url("https://github.com/mrpowers-io/spark-fast-tests"), "git@github.com:MrPowers/spark-fast-tests.git"))
 
 updateOptions := updateOptions.value.withLatestSnapshots(false)
 
-publishMavenStyle := true
-
-publishTo := sonatypePublishToBundle.value
-
-Global / useGpgPinentry := true
\ No newline at end of file
+import xerial.sbt.Sonatype.sonatypeCentralHost
+ThisBuild / sonatypeCredentialHost := sonatypeCentralHost
```

Comment:
@SemyonSinchenko @alfonsorr @MrPowers Either this if we have sonatypeCentral account or 

```scala
ThisBuild / sonatypeCredentialHost := "s01.oss.sonatype.org"
sonatypeRepository := "https://s01.oss.sonatype.org/service/local"
```

Reply:
If @MrPowers didn't create a new account, you don't need to configure the host with sbt-ci-release because he has an account in the old server and we have to publish there.

Reply:
@alfonsorr sure let's me remove this part. I probably going to just keep the generic part and leave the specific setup to whoever have control to these settings